.standard.yml

```
@@ -0,0 +1,3 @@
+# For available configuration options, see:
+#   https://github.com/testdouble/standard
+ruby_version: 2.5
```

```suggestion
ruby_version: 2.5

ignore:
  - '**/*':
      - Style/TrailingCommaInArguments
      - Style/TrailingCommaInArrayLiteral
      - Style/TrailingCommaInHashLiteral
```
I think this is the closest we can get to it 😝

---

Interesting... I'd be happy to go with this also, I do like leaving the trailing commas.

I guess let's see what the rest of the team thinks? I'm happy to apply the change, but want to see if we can get consensus :)